packages/calypso-analytics/README.md

```diff
@@ -0,0 +1,50 @@
+Analytics: Tracks
+=================
+
+Automatticians may refer to internal documentation for more information about Tracks.
```

Comment:
You can grab P2 shorthands for field guide pages, too. I think that would be helpful. Totes not a blocker for merging. :-)